tags.json

```
@@ -1410,6 +1410,9 @@
     "statistics",
     "diagram"
   ],
+  "pin": [
+    "pin"
```

An extra tag with the same content as the name is useless.

```suggestion
    "save",
    "map",
    "lock",
    "fix"
```